socnet/templates/messenger/messages_search.html

```
@@ -7,34 +7,34 @@
   {% for message, interlocutor in messages_ %}
     <a
-      href="{% url 'messenger:chat' interlocutor.username %}#msg{{ message.pk }}"
       class="d-flex align-items-center list-group-item list-group-item-action"
+      href="{% url 'messenger:chat' interlocutor.username %}#msg{{ message.pk }}"
     >
       <div
         class="avatar-thumbnail d-flex align-items-center justify-content-center me-2"
       >
         {% if interlocutor.image %}
-          <img src="{{ interlocutor.image.url }}" loading="lazy" class="rounded" />
+          <img class="rounded" loading="lazy" src="{{ interlocutor.image.url }}" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)